vaadin-select-flow-demo/src/main/java/com/vaadin/flow/component/select/entity/Country.java

```diff
@@ -0,0 +1,44 @@
+package com.vaadin.flow.component.select.entity;
+
+import java.util.Objects;
+
+public class Country {
```

Comment:
To place this new model classes (and possibly the existing ones) to card so as to have it available for user on VCOM `Select` component examples page. `GridDemo` in `vaadin-grid-flow` can be used as an example.